### ISSUE TITLE
should not depend on "jupyter" location to determine whether use suedo

### ISSUE THREAD

**Issue by clouds56** [akabe/ocaml-jupyter#68]
e.g. in mac with homebrew, `jupyter` locates at `/usr/local/bin/jupyter` while you do not require sudo permission to write to `/usr/local` (and should not)

I think that instead of install manage kernel via `opam`, I prefer just write a warning log like
```
please run 'jupyter kernelspec install [--user] "$datadir"' to install the kernel
```

**Comment by akabe** [comment]
Hmm, I see. I want to install the kernel automatically, but it is hard due to various OS as you said.
I will fix for preventing management of kernel and changing the opam message.
Thanks.

**Comment by akabe** [comment]
I fixed: PR #71
